src/sbt-test/kotlin/basic-android/project/android.sbt

```diff
@@ -1 +1 @@
-addSbtPlugin("org.scala-android" % "sbt-android" % "1.6.15")
+addSbtPlugin("org.scala-android" % "sbt-android" % "2.0.0-SNAPSHOT")
```

Comment:
This version of the android plugin (which uses sbt 1.x) can be built from here https://github.com/jeffboutotte/sbt-android/tree/sbt-1 .  Some unit tests are still failing for that plugin on that branch, so it's not quite ready, but the tests for this plugin pass locally with the updated version of `sbt-android`.

Reply:
Yeah, just won't be usable for sbt-android for now, I haven't had time to work on it--just non-android kotlin

Reply:
No worries - thanks for merging! I may keep pulling the thread on the android side when I find some free time.